src/Symfony/Component/Validator/Tests/Validator/RecursiveValidator2Dot5ApiTest.php

```diff
@@ -29,4 +30,28 @@ protected function createValidator(MetadataFactoryInterface $metadataFactory, ar
 
         return new RecursiveValidator($contextFactory, $metadataFactory, $validatorFactory, $objectInitializers);
     }
+
+    public function testEmptyGroupsArrayDoesNotTriggerDeprecation()
```

Comment:
I was wondering if this test method should still exists in 3.0? If not, then I'd say a legacy tag should be added.